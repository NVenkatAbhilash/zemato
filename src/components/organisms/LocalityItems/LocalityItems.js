import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { Grid } from "@material-ui/core";
import LocalityViewItem from "../../molecules/LocalityViewItem/LocalityViewItem";
import { Link } from "react-router-dom";

const useStyles = makeStyles({
  root: {
    padding: "40px",
  },
});

const LocalityItems = (props) => {
  const classes = useStyles();

  return (
    <Grid container className={classes.root} spacing={2} {...props}>
      {viewType.map((value, index) => {
        return (
          <Grid item md={4} sm={6} xs={12} key={index}>
            <Link to="/viewType/delivery" style={{ textDecoration: "inherit" }}>
              <LocalityViewItem {...value} ></LocalityViewItem>
            </Link>
          </Grid>
        );
      })}
    </Grid>
  );
};

const viewType = [
  {
    placesCount: 363,
    placeName: "Jubilee Hills",
  },
  {
    placesCount: 84,
    placeName: "Gachibowli",
  },
  {
    placesCount: 264,
    placeName: "Hitech City",
  },
  {
    placesCount: 93,
    placeName: "Madhapur",
  },
  {
    placesCount: 1153,
    placeName: "Kukatpally",
  },
  {
    placesCount: 166,
    placeName: "Begumpet",
  },
  {
    placesCount: 261,
    placeName: "Tolichowki",
  },
  {
    placesCount: 231,
    placeName: "Ameerpet",
  },
  {
    placesCount: 87,
    placeName: "Film Nagar",
  },
  {
    placesCount: 122,
    placeName: "Paradise Circle",
  },
  {
    placesCount: 48,
    placeName: "Necklace Road",
  },
  {
    placesCount: 363,
    placeName: "Jubilee Hills",
  },
  {
    placesCount: 84,
    placeName: "Gachibowli",
  },
  {
    placesCount: 264,
    placeName: "Hitech City",
  },
  {
    placesCount: 93,
    placeName: "Madhapur",
  },
  {
    placesCount: 1153,
    placeName: "Kukatpally",
  },
  {
    placesCount: 166,
    placeName: "Begumpet",
  },
  {
    placesCount: 261,
    placeName: "Tolichowki",
  },
];

export default LocalityItems;
