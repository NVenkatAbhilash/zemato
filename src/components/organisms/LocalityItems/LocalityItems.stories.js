import React from "react";
import LocalityItems from "./LocalityItems";

export default {
    title: "organisms/LocalityItems",
    component: LocalityItems,
}


const Template = ({...args})=>(
    <LocalityItems {...args} />
);


export const Primary = Template.bind({});
Primary.args = {
}
