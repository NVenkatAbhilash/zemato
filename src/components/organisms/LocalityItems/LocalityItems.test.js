import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import "@testing-library/jest-dom";
import LocalityItems from "./LocalityItems";
import renderer from "react-test-renderer";

it("renders LocalityItems snapshot correctly", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <LocalityItems />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders LocalityItems without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <BrowserRouter>
      <LocalityItems />
    </BrowserRouter>,
    div
  );
});
