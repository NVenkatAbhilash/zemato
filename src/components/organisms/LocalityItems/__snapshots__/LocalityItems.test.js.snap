// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LocalityItems snapshot correctly 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-spacing-xs-2"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Jubilee Hills
             (
            363
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Gachibowli
             (
            84
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Hitech City
             (
            264
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Madhapur
             (
            93
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Kukatpally
             (
            1153
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Begumpet
             (
            166
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Tolichowki
             (
            261
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Ameerpet
             (
            231
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Film Nagar
             (
            87
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Paradise Circle
             (
            122
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Necklace Road
             (
            48
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Jubilee Hills
             (
            363
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Gachibowli
             (
            84
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Hitech City
             (
            264
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Madhapur
             (
            93
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Kukatpally
             (
            1153
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Begumpet
             (
            166
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
  >
    <a
      href="/viewType/delivery"
      onClick={[Function]}
      style={
        Object {
          "textDecoration": "inherit",
        }
      }
    >
      <div
        className="MuiGrid-root makeStyles-root-2 MuiGrid-container MuiGrid-align-items-xs-center"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6"
            data-testid="text"
          >
            Tolichowki
             (
            261
             places)
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root makeStyles-arrowIcon-3"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
</div>
`;
